packages/iTwinUI-react/src/core/Wizard/Wizard.tsx

```diff
@@ -2,11 +2,11 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import cx from 'classnames';
 import React from 'react';
-import { useTheme } from '../utils';
 import '@itwin/itwinui-css/css/stepper.css';
-import { Step } from './Step';
+import '@itwin/itwinui-css/css/workflow-diagram.css';
```

Comment:
CSS imports are not needed as they should be inside Stepper and Workflow components.

Reply:
Removed CSS imports